frontend/src/App.js

```diff
@@ -1,12 +1,20 @@
-import { BrowserRouter as Router } from "react-router-dom";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./index.css";
 import Navbar from "./Components/Navbar";
 import MiniNav from "./Components/MiniNavbar";
+import Home from "./Pages/Home";
+import Login from "./Pages/Login";
+import Signup from "./Pages/Signup";
 function App() {
   return (
     <Router>
       <MiniNav />
       <Navbar />
+      <Routes>
+        <Route path="/" exact={true} element={<Home />} />
```

Comment:
`exact` is no more used in `react-router-v6`

Reply:
Yes just got to know about that.